pkg/lib/authn/otp/sender.go

```diff
@@ -70,6 +72,16 @@ func (s *MessageSender) SendEmail(email string, opts SendOptions) error {
 	}
 	data.Email = email
 
+	if opts.OTPMode == OTPModeMagicLink {
+		url := s.Endpoints.MagicLinkVerificationEndpointURL()
+		query := url.Query()
+		query.Set("token", data.Code)
+		query.Set("target", data.Email)
```

Comment:
Please be aware that we should not include any PII in query parameters. That information may leave in the browser history/server log...etc. 

ref: https://owasp.org/www-community/vulnerabilities/Information_exposure_through_query_strings_in_url